pandas/io/formats/excel.py

```diff
@@ -160,6 +160,22 @@ class CSSToExcelConverter:
         "fantasy": 5,  # decorative
     }
 
+    BORDER_STYLE_MAP = {
+        style.lower(): style
```

Comment:
Hmm why did this become case insensitive now? I though before it should have been explicit  (which was better IMO)?

Reply:
`CSSResolver.parse` applies a lower-case conversion to all CSS properties and values. There's a TODO note from 6 years ago to not lowercase case-sensitive portions, but it looks like that still hasn't been implemented.

Reply:
Given that CSS is case-insensitive already, I don't think the HTML-based formatters need the case-lowering since the browser will interpret "solid" and "SOLID" the same. I think that means we could move case translation out of the general CSS classes and into the Excel formatters and more easily control for case-sensitive values.